bigtable/api/Snippets/Reads/ReadFilter.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) 2020 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+// [START bigtable_reads_filter]
+
+using Google.Cloud.Bigtable.Common.V2;
+using Google.Cloud.Bigtable.V2;
+using System.Threading.Tasks;
+
+namespace Reads
+{
+    public class ReadFilter
+    {
+        /// <summary>
+        /// Reads using a filter from an existing table.
+        ///</summary>
+        /// <param name="projectId">Your Google Cloud Project ID.</param>
+        /// <param name="instanceId">Your Google Cloud Bigtable Instance ID.</param>
+        /// <param name="tableId">Your Google Cloud Bigtable table ID.</param>
+        public static async Task<string> BigtableReadFilter(
```

Comment:
I see that this is the same code that you have on the helper ReadFilter class (except that the helper receives a given filter). And I see that this is in a sample so ultimately users can find this sample to "complete the picture" of the filter ones. Still, I wouldn't like to make it hard for users to undersntand a single sample, there's no easy way for jumping from one sample to another, etc. Even though we keep this sample more or less as it is, I still think all samples should be self contained.
Maybe in the filter samples just print thet amount of rows returned or the information for the first row returned, etc. Something that shows that you get an async stream when filtering is enough. Let me know if you want to chat about this.